react/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ErrorInfo, FunctionComponent } from 'react'
+import ExtensionPointError from './ExtensionPointError'
+import { useRuntime } from './RenderContext'
+
+interface Props {
+  runtime: any
```

Comment:
there is a `RenderRuntime` type

Reply:
@jgfidelis good catch, left this as a WIP and forgot to update